### DISCUSSION TITLE
Integrazione con la building automation di Idrosistemi

### DISCUSSION THREAD

**Discussion by rallo90** [indomus/forum#2504] [HUB: Home Assistant]
Buongiorno!
utilizzo da un paio di anni home assistant insieme a nabu casa, componenti android e tra poco anche shelly.
In questi giorni mi stanno installando la building automation di Idrosistemi per la regolazione del clima, della vmc e la misurazione dei consumi domestici.
In rete ho trovato pochissime info a riguardo e tanto meno modalità di integrazione con home assistant. Potete aiutarmi?
grazie
ciao!

**Comment by indomus** [comment]
> Buongiorno! utilizzo da un paio di anni home assistant insieme a nabu casa, componenti android e tra poco anche shelly. In questi giorni mi stanno installando la building automation di Idrosistemi per la regolazione del clima, della vmc e la misurazione dei consumi domestici. In rete ho trovato pochissime info a riguardo e tanto meno modalità di integrazione con home assistant. Potete aiutarmi? grazie ciao!

Hai sentito in primis Idrosistemi per capire che interfacce offrano?

**Reply by rallo90** [reply]
Buongiorno! grazie per il consiglio, gli ho scritto e ho ricevuto la seguente risposta. Ora come mi consigliate di procedere?

abbiamo l'interfaccia konnex - modbus rtu - modbus eth

1- KNX
Gateway per interfacciamento del Sistema con domotica KNX
È predisposto per il fissaggio in quadro elettrico, su guida a norma DIN (4 moduli).
Il sistema di supervisione a cui il gateway viene collegato deve avere protocollo KNX
(konnex). La parte di connessione al bus KNX è implementata tramite un modulo di
comunicazione certificato. Il gateway REG-KNX mette a disposizione 250 datapoint,
che sono liberamente configurabili da chi configura il sistema domotica KNX

2-MBS-ETH
Interfaccia per supervisione Sistema Modbus over TCP/IP
L’interfaccia per supervisione di un sistema ModBus over TCP/IP è predisposta per il fissaggio in quadro elettrico, su guida a norma DIN (2 moduli). Il sistema di supervisione a cui l’interfaccia viene collegata deve avere protocollo
ModBus TCP/IP. Quando viene consegnato il modulo ha preimpostato l’indirizzo
192.168.127.254, per modificarlo fare riferimento al manuale completo.

3- MBS-RTU
Interfaccia per supervisione Sistema Modbus over 485 (protocollo RTU)
L’interfaccia per supervisione di un sistema ModBus over 485 (protocollo RTU) è
predisposta per il fissaggio in quadro elettrico, su guida a norma DIN (2 moduli).
Il sistema di supervisione a cui l’interfaccia viene collegata deve avere protocollo
ModBus RTU (RS-485). L’interfaccia permette ad un sistema di supervisione di
terze parti di modificare e leggere i parametri del sistema REG, che viene visto
come dispositivo slave.

**Reply by indomus** [reply]
puoi usare:

https://www.home-assistant.io/integrations/knx/
https://www.home-assistant.io/integrations/modbus/

**Comment by rallo90** [comment]
Ok grazie, me ne consigliate uno in particolare?

**Reply by indomus** [reply]
KNX dovrebbe essere più agevole

**Reply by rallo90** [reply]
Per quanto sono riuscito a capire ho la necessità di acquistare un Interfaccia KNX IP da connettere al raspberry e all'interfaccia fornita da Idrosistemi, è corretto? Avete un'Interfaccia KNX IP in particolare da suggerirmi?
grazie
ciao

**Comment by FuL75** [comment]
Ciao @rallo90 , mi ricollego a questo post perché anch'io ho un sistema come il tuo e stavo cercando in rete un po' di informazioni ed anche a me hanno detto quello che hanno detto a te dalla idrosistemi. Sei riuscito a portare a termine il tuo progetto poi?
Io ho fatto diverse prove anche con Scraper per vedere se riuscivo ad interagire con il supervisore vi browser ma non riesco ad accederci.

**Reply by rallo90** [reply]
Ciao FuL75, no mi dispiace ma ho abbandonato il progetto perché avevo altre cose da fare e sono ancora fermo a quello che vedi sopra.
Dovremmo forse sentire qualcuno che mastica un pò di knx? ho provato anche a scrivere su plc forum ma senza ricevere risposta.

**Reply by rallo90** [reply]
forse dovremmo acquistare il componente della idrosistemi per connettere in lan al raspberry tramite in componente come questo? weinzierl knx ip baos 774
andremmo a spendere un bel pò

**Reply by FuL75** [reply]
Ciao, anch'io ho abbandonato il progetto per il momento, avendo scritto a idrosistemi che mi ha sconsigliato. Quello che sono riuscito a fare, almeno finora, è stato quello di montare un modulino Modbus nella VMC per poterla gestire via Home Assistant, almeno cambiare la portata e altri parametri che altrimenti dalla supervisione della PDC non si poteva fare.
In generale non mi trovo male con il "loro" sistema, la cosa che però non sopporto è che quando "sente" il surplus fotovoltaico fa partire il COMFORT anche se a dare il surplus sono solo le batterie. E ovviamente me le scarica....

**Comment by rallo90** [comment]
A me non da questo problema, mi sono fatto impostare elettra in modo che
faccia lo switch su comfort soltanto quando c'è effettivamente immissione
in rete

Il Mer 24 Gen 2024, 17:05 FuL75 ***@***.***> ha scritto:

**Reply by FuL75** [reply]
E bravo. è esattamente quello che avevo chiesto io quando gli ho scritto a giugno!!! Però non me l'hanno ancora modificato!!
Con chi hai parlato, se posso chiedertelo? Di che zona sei?

**Comment by rallo90** [comment]
Io sono di Arezzo. Il termotecnico che mi ha seguito è anche un credo
agente della idrosistemi e mi sono sempre interfacciato con lui, purtroppo
non mi ha dato i contatti diretti. Dovresti sentire chi ha venduto il
sistema al tuo elettricista/idraulico

Il Mer 24 Gen 2024, 17:49 FuL75 ***@***.***> ha scritto:

**Reply by FuL75** [reply]
Ah, se sei di Arezzo sicuramente ti avrà seguito Ferdinando. Ha seguito anche me e mi sono trovato bene.

**Reply by FuL75** [reply]
Vai, gli ho riscritto e dice me l'hanno modificato così: "per far partire elettra devono esserci almeno 500 watt in immissione e si disabilita non appena tocca lo 0 di immissione"

**Comment by rallo90** [comment]
Si mi ha seguito lui. Per quanto possibile, dato il periodo, mi sono
trovato bene. Io ho la pompa di calore un po sovradimensionata e per
limitare il problema degli on/off voglio chiedere se sia possibile far
partire l'integrazione della pompa di calore alla vmc ogni volta che almeno
un circuito del radiante è aperto.

Il Gio 25 Gen 2024, 17:55 FuL75 ***@***.***> ha scritto:

**Comment by rallo90** [comment]
Ciao, sono riuscito a farmi fare la modifica che ti dicevo in modo da avere
sempre l'integrazione tramite la vmc ogni volta che parte il radiante.
Adesso sto facendo lo switch su estate. Te quale temperatura di mandata
tieni per far deumidificare bene?


Il Dom 28 Gen 2024, 16:48 Francesco Ralli ***@***.***> ha scritto:

**Reply by FuL75** [reply]
Ciao, la temperatura la gestisce direttamente Elettra, io non la posso cambiare, però se non ricordo male sta intorno ai 12 gradi. Appena switcho su estate ti so dire meglio.
Io uso il valore di quella temperatura per attivare un booleano che mi dice se è accesa la deumidifica o no, perchè la mia VMC purtroppo non mi esporta su modbus lo stato di funzionamento, e nemmeno lo comunica a Elettra

**Comment by rallo90** [comment]
Io purtroppo non ho la pompa di calore idrosistemi ma ho la mitsubishi non
modificata e quindi imposto la temperatura a mano e regolo manualmente
anche la valvola che miscela l acqua prima di entrare nel radiante.
Fammi sapere la temperatura allora grazie

Il Mer 15 Mag 2024, 08:57 FuL75 ***@***.***> ha scritto:

**Reply by FuL75** [reply]
Eccomi, non mi ero dimenticato, oggi ho fatto lo switch. Qui sotto uno screenshot delle varie temperature della VMC con la deumidifica accesa:
![image](https://github.com/indomus/forum/assets/124737429/372c0858-58c0-401a-b7d8-9140cd746c5f)

**Comment by rallo90** [comment]
Ciao grazie mille.
Da dove li leggi questi dati? io non li ho e e penso che il motivo sia
perché non ho la pdc idrosistemi.
 sei riuscito a visualizzare i dati da home assistant?

Il Mer 29 Mag 2024, 16:35 FuL75 ***@***.***> ha scritto:

**Reply by FuL75** [reply]
questa è una foto di Home Assistant, i dati li estraggo via Modbus direttamente dalla VMC, che è Idrosistemi anche lei.

**Comment by rallo90** [comment]
Mi diresti come hai fatto?
Riesci a vedere anche temperatura e umidità delle stanze?

Il Gio 30 Mag 2024, 09:52 FuL75 ***@***.***> ha scritto:

**Reply by FuL75** [reply]
quelle purtroppo no, dovrei collegarmi direttamente a Elettra con un adattatore apposito ma la porta è già usata per la remotazione dul cloud, quindi niente. Ho provato in tutti i modi con l'integrazione scrape a tirare giù i valori dal portale della supervisione ma non c'è modo!!
Per la VMC invece è stato molto semplice, con un modulino Modbus/Ethernet collegato alla porta RS485 e gli indirizzi Modbus della VMC che ho trovato sul manuale
